opennms-doc/guide-admin/src/asciidoc/text/poller/monitors/PageSequenceMonitor.adoc

```diff
@@ -0,0 +1,310 @@
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[poller-page-sequence-monitor]]
+==== PageSequenceMonitor
+
+The _PageSequenceMonitor_ (PSM) allows to monitor web applications.
+This monitor has several configuration options regarding _IPv4_, _IPv6_ and how to deal with name resolution.
+To add flexibility the node label and IP address can be passed as variable into the monitor.
+This allows running the monitor with node dependent configuration.
+Beyond testing a web application with a single _URL_ it allows also to test a path through a web application.
+A test path through an web application can look like this:
+
+. login to a certain web application
+. Execute an action while being logged in
+. Log off
+
+The service is considered as _up_ if all this is working ok.
+If there's an error somewhere, your application will need attention and the service changes the state to _down_.
+
+[[poller-page-sequence-monitor-facts]]
+===== Monitor facts
+
+[options="autowidth"]
+|===
+| Class Name     | `org.opennms.netmgt.poller.monitors.PageSequenceMonitor`
+| Remote Enabled | true
+|===
+
+[[poller-page-sequence-monitor-configuration]]
+===== Configuration and Usage
+
+The configuration for this monitor consists several parts.
+First part is the overall configuration for `retries` and `timeouts`.
+These parameters are global for the whole path through the web application.
+
+.Configuration overview of the _PSM_
+image::poller/01_page-sequence-monitor-config.png[]
+
+The overall layout of the monitor configuration is more complex.
+Additional to the parameter keys it is possible to configure a page sequence containing a path through an web application.
+
+.Monitor parameters for the PageSequenceMonitor
+[options="header, autowidth"]
+|===
+| Parameter        | Description                                                                           | Required | Default value
+| `timeout`        | Time in milliseconds to wait before assuming that a packet has not responded          | optional | `3000`
+| `retry`          | The number of retries to attempt when a packet fails to respond in the given timeout  | optional | `0`
+| `strict-timeout` | ?? `true` or `false`                                                                  | optional | `false`
```

Comment:
@jeffgdotorg Do you have a description sentence for `strict-timeout` parameter setting?